packages/data-store/src/entities/contact/PartyTypeEntity.ts

```diff
@@ -14,6 +24,9 @@ export class PartyTypeEntity {
   @Column('simple-enum', { name: 'type', enum: PartyTypeEnum, nullable: false, unique: false })
   type!: PartyTypeEnum
 
+  @Column('simple-enum', { name: 'origin', enum: PartyOrigin, nullable: true, unique: false })
```

Comment:
I think we discussed in the sprint start to have a default (external) and make it non-nullable

Reply:
Made it non-nullable with external as default